server/controllers/notificationController.ts

```diff
@@ -76,7 +76,7 @@ export default class NotificationController {
     }
     await this.notificationService.updateSnooze(
       user.token,
-      NotificationController.getFormattedFutureDate(pauseUnit, pauseValue),
+      NotificationController.getFormattedFutureDate(pauseUnit, Number(pauseValue)),
```

Comment:
latest date-fns returned completely the wrong date passing in a string instead here